scene/gui/text_edit.h

```diff
@@ -809,6 +810,9 @@ class TextEdit : public Control {
 	void set_caret_blink_interval(const float p_interval);
 	float get_caret_blink_interval() const;
 
+	bool is_caret_on_readonly();
+	void set_caret_on_readonly(bool p_value);
```

Comment:
Setter always goes first, would also suggest calling it the follow to align with the rest of the class:
`[set_draw|is_drawing]_caret_when_readonly_enabled(bool p_enabled)`